docs/Homebrew-and-Python.md

```diff
@@ -85,13 +87,21 @@ Since the system Python may not know which compiler flags to set when building b
 CFLAGS="-I$(brew --prefix)/include" LDFLAGS="-L$(brew --prefix)/lib" pip install <package>
 ```
 
-## Virtualenv
-
 **Warning!** When you `brew install` formulae that provide Python bindings, you should **not be in an active virtual environment.**
 
-Activate the virtualenv *after* you've brewed, or brew in a fresh terminal window. This will ensure Python modules are installed into Homebrew's `site-packages` and *not* into that of the virtual environment.
+Activate the virtualenv *after* you have installed your package with brew, or install brew's packages in a fresh terminal window. This will ensure Python modules are installed into Homebrew's `site-packages` and *not* into that of the virtual environment.
+
+## PEP688 (Python@3.12) and virtualenvs
+
+Starting with Python@3.12, Homebrew follows PEP688:
+https://peps.python.org/pep-0668/#marking-an-interpreter-as-using-an-external-package-manager
+
+To install Python packages system-wide, try `brew install xyz`, where xyz is the package you are trying to install.
+
+If you wish to install a non-brew-packaged Python package, create a virtual environment using `python3 -m venv path/to/venv`.
+Then use `path/to/venv/bin/python` and `path/to/venv/bin/pip`.
 
-Virtualenv has a `--system-site-packages` switch to allow "global" (i.e. Homebrew's) `site-packages` to be accessible from within the virtualenv.
+If you wish to install a non-brew packaged Python application, it may be easiest to use `pipx install xyz`, which will manage a virtual environment for you. Make sure you have `pipx` installed.
```

Comment:
```suggestion
If you wish to install a non-brew packaged Python application, it may be easiest to use `pipx install xyz`, which will manage a virtual environment for you.
You can install `pipx` using Homebrew, using the command `brew install pipx`.
When you use `pipx` to install a Python application, it will always use a virtual environment for you, even if `pipx` was installed via Homebrew.
```

Reply:
Yeh, once sentence per line would be nice.